add a better clarification on how sysvar changes "propagate" (#5562)
pingcap/docs: system-variables.md

> **Note:**
>

> > TiDB differs from MySQL in that `GLOBAL` scoped variables **persist** through TiDB server restarts. Changes to `GLOBAL` variables may take up to 2 seconds to be effective, including the TiDB server where the `SET` statement was issued. See [TiDB #14531](https://github.com/pingcap/tidb/issues/14531).

Reviewer: Would this be clearer without changing any meaning?
```suggestion
> TiDB differs from MySQL in that `GLOBAL` scoped variables **persist** through TiDB server restarts. Changes to `GLOBAL` variables might take up to 2 seconds to be effective, including on the TiDB server where the changes are made. See [TiDB #14531](https://github.com/pingcap/tidb/issues/14531) for details.
```
